projects/exui-common-lib/src/lib/services/address/address-parser.ts

```diff
@@ -0,0 +1,81 @@
+import { AddressModel } from '../../models';
+import { AddressType } from './address-type.enum';
+
+/**
+ * Moving all this logic here into Address Parser class, so that it
+ * will be easier for us when we move this parsing logic to into
+ * `Shim` java service.
+ */
+ export class AddressParser {
+  public parse(address: any) {
+    const classification = `${address.CLASSIFICATION_CODE}`;
+    const addressModel = new AddressModel();
+    addressModel.addressLine1 = this.parseAddressLine1(classification, address);
+    addressModel.addressLine2 = this.parseAddressLine2(classification, address);
+    addressModel.addressLine3 = this.parseAddressLine3(classification, address);
+    addressModel.postCode = address.POSTCODE;
+    addressModel.postTown = address.POST_TOWN;
+    addressModel.country = AddressType.UK;
+    return addressModel;
+  }
+
+  private parseAddressLine1(classification: string, address: any) {
```

Comment:
@johnbenjamin-hmcts - missing return type

Reply:
@Daniel-Lam - I have updated the code.